bundles/com.espressif.idf.core/src/com/espressif/idf/core/build/IDFBuildConfiguration.java

```diff
@@ -431,8 +431,13 @@ private void runCmakeCommand(IConsole console, IProgressMonitor monitor, IProjec
 
 		infoStream.write(String.format(Messages.CMakeBuildConfiguration_Configuring, buildDir));
 
+		IDFEnvironmentVariables envVariables = new IDFEnvironmentVariables();
+		String pythonPath = envVariables.getEnvValue(IDFEnvironmentVariables.IDF_PYTHON_ENV_PATH);
 		List<String> command = new ArrayList<>();
-
+		if (!StringUtil.isEmpty(pythonPath))
+		{
+			command.add(pythonPath);
+		}
```

Comment:
Hi @kolipakakondal. Is this correct? Would this run CMake as a Python script?
I think the proper way is to call cmake directly and pass the Python path using the -DPYTHON=... argument instead.

Also, do we need to include the following arguments as well?
```
command.add("-DPYTHON_DEPS_CHECKED=1");  // Skip Python dependency checks
command.add("-DPYTHON=" + IDFUtil.getIDFPythonEnvPath()); // Use specific Python path
command.add("-DESP_PLATFORM=1"); // Enable ESP-IDF platform-specific behavior
```